### ISSUE TITLE
update orgmode syntax: #+begin_export backend vs. #+begin_backend

### ISSUE THREAD

**Issue by jwhendy** [getnikola/plugins#228]
Hello,

Some time ago, Org adjusted the syntax for exported language blocks. I'm not positive when... I actually thought it was back with the [release of Org-mode 8.0](http://orgmode.org/worg/org-8.0.html), but these [release notes](http://orgmode.org/cgit.cgi/org-mode.git/plain/etc/ORG-NEWS) suggest it might be 9.1 that brought the change. I've been on git forever, so I'm not sure.

```
Export blocks are explicitly marked as such at the syntax level to
disambiguate their parsing from special blocks.  The new syntax is

#+BEGIN_EXPORT backend
...
#+END_EXPORT

instead of

#+BEGIN_backend
...
#+END_backend
```
Here is my `.org`. test file:

```
#+begin_comment 
.. title: index
.. slug: index
.. description: org test
#+end_comment

* section

Here is some raw html code:

#+begin_export html 
bold text: <strong>test</strong>
#+end_export

#+begin_html 
bold text: <strong>test</strong>
#+end_html
```

I left `plugin/init.el` alone except for uncommenting the load path:
```
;; Load org-mode
;; Requires org-mode v8.x

;; Uncomment these lines and change the path to your org source to
;; add use it.
(let* ((org-lisp-dir "~/.elisp/org/lisp"))
  (when (file-directory-p org-lisp-dir)
      (add-to-list 'load-path org-lisp-dir)
      (require 'org)))
```
Just to confirm, this is definitely where my orgmode lives:
```
M-x org-version
Org mode version 9.0.5 (release_9.0.5-288-g4caad0 @ /home/jwhendy/.elisp/org/lisp/)
```

The respective sections get exported like so, showing that `#+begin/end_export html` is not picked up (first instance), but `#+begin/end_html` is (second, rendered instance).

```
<div class="export">
<p>
bold text: &lt;strong&gt;test&lt;/strong&gt;
</p>
</div>

bold text: <strong>test</strong>
```
Screenshot of served webpage:

![2017-05-15_000148](https://cloud.githubusercontent.com/assets/379357/26043289/bc95ec16-3901-11e7-8e1d-887c5a6e6a40.png)

Org does the opposite using `C-c C-e h` and one of the `html` options, rendering the correct syntax and botching the deprecated variant:
```
bold text: <strong>test</strong>

<div class="html">
<p>
bold text: &lt;strong&gt;test&lt;/strong&gt;
</p>
```
![2017-05-15_002116](https://cloud.githubusercontent.com/assets/379357/26043647/6fd02b78-3904-11e7-9e93-8874d625d5a9.png)

Let me know if there's something I should check for on my end, and thanks for taking a look!

**Comment by punchagan** [comment]
@jwhendy I'm not sure what this issue exactly expects us to do on the Nikola plugin end. Org-mode's release notes provide [a function to fix existing files](http://orgmode.org/Changes.html#orgf3f9c91), and you should probably run that on all your Nikola post files. I don't think such a fix should be implemented in the plugin. If you are convinced otherwise, feel free to re-open the issue and we can discuss to see what can be done.

**Comment by punchagan** [comment]
You could use [this script](https://gist.github.com/punchagan/ead378180ad702b616fd74c01e73d063) to fix the posts in your blog, if you like. I tested it on my own blog, and it seems to work fine. Please create a backup copy of your posts before running this script, though.

**Comment by jwhendy** [comment]
After typing up a lengthy response, I think I found the issue but don't understand.

My entire issue above was that I'm *already* using the new syntax, but Nikola wasn't recognizing it. Take a look at the reproducible example: I showed that the first (correct) syntax rendered verbatim whereas the old/deprecated syntax produced the desired result (bold text).

Org's output was exactly the opposite, also shown.

---

After digging around some more, I found that if I use `emacs -Q`, then my org-mode output matches Nikola. So, I think the issue is not having my git installation of org found; instead it's using the version shipped with emacs. I posted my config above; any reason this wouldn't work?

Here it is again:
```
$ cat ~/Desktop/mysite/plugins/orgmode/init.el
(let* ((org-lisp-dir "~/.elisp/org/lisp"))
  (when (file-directory-p org-lisp-dir)
      (add-to-list 'load-path org-lisp-dir)
      (require 'org)))

(require 'ox-html)
```

---

After even *more* fiddling, I'm at a loss. I started getting errors about a bad emacs config when I tried to use a `#+begin_src elisp` block to output the result of `M-x org-version` during the export (to confirm which version of org Nikola used). Then I couldn't make it go away!

Wiped the plugins dir, reinstalled, and now things are working fine. I'm at a complete loss, but I changed so much during my hunt I couldn't tell you what it was. I even diffed my pasted plugin `init.el` from yesterday with the one that's now working and they are identical.

Sorry for the noise, and thanks for taking a look.
